package.json

```diff
@@ -28,8 +26,7 @@
   },
   "lint-staged": {
     "*.{js,jsx,ts,tsx}": [
-      "yarn lint:precommit --fix",
-      "prettier --write",
+      "yarn lint:ts --fix",
       "git add"
     ]
   },
```

Comment:
This appears to be unused, so you can remove it

Reply:
done